ci/ansible/roles/pulp-crane/files/pulp_crane_el6.conf

```diff
@@ -0,0 +1,15 @@
+# source:
+# https://raw.githubusercontent.com/pulp/crane/3c657fe53020b804ce60d907a82b4fb19aebe532/deployment/apache22.conf
+# Use this config with Apache 2.2 or earlier
+<VirtualHost *>
```

Comment:
Did you test whether Pulp+Crane works when accessed from a Docker client? I strongly suspect that this config file is broken, for a couple reasons:

* This `VirtualHost` directive listens on port 80, not port 5000.
* The `Location` directive listens only for Docker requests that originate from localhost.

The sample Apache 2.4 configuration file provided by the Crane repository is basically broken for production purposes by default, and we don't use it here in pulp_packaging. The same may be true for the sample Apache 2.2 configuration file.